staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -333,9 +333,12 @@ type ListOptions struct {
 	// add, update, and remove notifications. Specify resourceVersion.
 	// +optional
 	Watch bool `json:"watch,omitempty" protobuf:"varint,3,opt,name=watch"`
-	// When specified with a watch call, shows changes that occur after that particular version of a resource.
+	// When specified with a watch call:
+	// shows changes that occur after that particular version of a resource.
+	// If it's 0, then the result will be current objects directory defined by key in "ADDED" events
```

Comment:
suggest `If it's 0, the result is an "ADDED" event for every existing object followed by events for changes that occur after the watch was established.`

Reply:
+1 to this, plus also include the leading `-` on these bullet points for clarity, or convert to prose (but `-` bullet points are consistent with the list section).

Reply:
fwiw, the bullets don't render great in contexts like https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#list-62

Reply:
"current objects directory defined by key" doesn't have a lot of meaning for a REST API call